### ISSUE TITLE
[Feature Request] Detect active player or allow setting player priority

### ISSUE THREAD

**Issue by renyuneyun** [osdlyrics/osdlyrics#127]
I'm not sure if this would be useful for other people as well, but it will be a much-needed feature for my setting.
I use MPD as my normal player, which also reads my playlists. At times, I will also temporarily listen to specific music through another player, e.g. clementine, which has a GUI and will not tamper with my MPD play status.

However, osdlyrics will not correctly pick up the player in that scenario. It will always pick up MPD. Indeed, MPD is running as daemon, but it is simply not playing (paused).

Thus, it would be really useful to detect the "active" players.
Otherwise, allow setting the player priority will also solve this issue (either recording history and drag, or string/regex match of player process name).

**Comment by kbsky** [comment]
Agreed, there's a lot of room for improvement in that area, I might end up looking into it one day but if anyone wants to contribute to OSD Lyrics that would be a great place to start.
